### PR TITLE
Update crystal version, fix port bindings, add more init methods

### PR DESCRIPTION
**Pull request by Vici37**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,5 +100,4 @@ require "docr"
 ## Contributors
 
 - [Tudor Marghidanu](https://github.com/marghidanu) - creator and maintainer
-
-[Contribution guidelines for this project](docs/CONTRIBUTING.md)
```
Comment (Vici37) [inline README.md:104]: Removed link to non-existent file

File: src/types/mount.cr [hunk 2]
```diff
@@ -42,20 +48,23 @@ module Docr::Types
...
-    property target : String?
-
     @[JSON::Field(key: "Source")]
     property source : String
 
     @[JSON::Field(key: "Type")]
     property type : String
 
+    @[JSON::Field(key: "Target")]
+    property target : String?
```
Comment (Vici37) [inline src/types/mount.cr:66]: Clustered and moved the non `?` types to the top of the class, which looks more like I moved `target` down :P

**Comment by Vici37** [comment]
Thanks for creating this crystal library! Looks like the most complete one I've seen, so using it for a toy project of mine that required interacting with docker directly, and starting with this one made that easier!

**Comment by marghidanu** [comment]
Thanks for the contribution, to be honest, I developed this quite fast so I knew that some features might be missing.

**Pull request merged**